Add new component and block templates (#39)
croct-tech/templates: templates/croct/ui/block/logo-cloud/README.md

# Introduction

> A logo cloud displays a collection of brand logos to showcase partnerships, integrations, clients, or media features.

Reviewer: ```suggestion
A logo cloud is commonly used to showcase partnerships, integrations, clients, or media features.
```
